app/javascript/components/supplemental_application/SupplementalApplicationPage.js

```diff
@@ -330,34 +310,27 @@ class SupplementalApplicationPage extends React.Component {
     }
 
     const context = {
+      ...this.state,
       assignSupplementalAppTouched: this.assignSupplementalAppTouched,
-      application: persistedApplication,
-      applicationMembers: getApplicationMembers(persistedApplication),
-      amis: amis,
-      amiCharts: amiCharts,
-      availableUnits: availableUnits,
-      statusHistory: statusHistory,
-      fileBaseUrl: fileBaseUrl,
-      loading: loading,
-      rentalAssistanceLoading: rentalAssistanceLoading,
       setLoading: this.setLoading,
       onSubmit: this.handleSaveApplication,
       onSavePreference: this.handleSavePreference,
-      confirmedPreferencesFailed: confirmedPreferencesFailed,
       onDismissError: this.handleDismissError,
-      statusModal: statusModal,
       openAddStatusCommentModal: this.openAddStatusCommentModal,
       openUpdateStatusModal: this.openUpdateStatusModal,
       handleStatusModalClose: this.handleStatusModalClose,
       handleStatusModalStatusChange: this.handleStatusModalStatusChange,
       handleStatusModalSubmit: this.handleStatusModalSubmit,
-      showNewRentalAssistancePanel: showNewRentalAssistancePanel,
-      showAddRentalAssistanceBtn: showAddRentalAssistanceBtn,
       hideAddRentalAssistanceBtn: this.hideAddRentalAssistanceBtn,
       handleOpenRentalAssistancePanel: this.handleOpenRentalAssistancePanel,
       handleCloseRentalAssistancePanel: this.handleCloseRentalAssistancePanel,
       handleSaveRentalAssistance: this.handleSaveRentalAssistance,
-      handleDeleteRentalAssistance: this.handleDeleteRentalAssistance
+      handleDeleteRentalAssistance: this.handleDeleteRentalAssistance,
+      application: persistedApplication,
```

Comment:
I wonder if there's a clearer way we can organize all of these, maybe alphabetical?

Reply:
Alphabetized